crypto/keys/keyring.go

```diff
@@ -419,29 +418,7 @@ func (kb keyringKeybase) Delete(name, _ string, _ bool) error {
 // The oldpass must be the current passphrase used for encryption, getNewpass is
 // a function to get the passphrase to permanently replace the current passphrase.
 func (kb keyringKeybase) Update(name, oldpass string, getNewpass func() (string, error)) error {
-	info, err := kb.Get(name)
```

Comment:
why is this a no-op now?

Reply:
`keyringKeybase.Update()` is already unused in keyring, see the `update` command (per-key password cannot be changed, the whole keyring is protected by a password in all implementations except for the `test` backend, which is password-less by design)